lxc/action.go

```diff
@@ -231,30 +290,13 @@ func (c *cmdAction) Run(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
-		for _, resource := range resources {
-			// We don't allow instance names with --all.
-			if resource.name != "" {
-				return fmt.Errorf(i18n.G("Both --all and instance name given"))
-			}
-
-			ctslist, err := resource.server.GetInstances(api.InstanceTypeAny)
-			if err != nil {
-				return err
-			}
+		if c.flagConsole != "" {
+			return fmt.Errorf(i18n.G("--console can't be used with --all"))
+		}
 
-			for _, ct := range ctslist {
-				switch cmd.Name() {
-				case "start":
-					if ct.StatusCode == api.Running {
-						continue
-					}
-				case "stop":
-					if ct.StatusCode == api.Stopped {
-						continue
-					}
-				}
-				names = append(names, fmt.Sprintf("%s:%s", resource.remote, ct.Name))
-			}
+		err = c.doActionAll(cmd.Name(), resources)
+		if err != nil {
+			return err
 		}
 	} else {
 		names = args
```

Comment:
If the user passes in multiple instance names (I.e. `lxc start thing1 thing2 thing3`), should this processing also happen on the backend? We designed this new endpoint focusing on the case when the user explicitly gives `--all`

Reply:
Nope, for now we'll just care about `--all`.